components/image-builder-bob/pkg/builder/config.go

```diff
@@ -67,60 +63,5 @@ func GetConfigFromEnv() (*Config, error) {
 		}
 	}
 
-	var authKey = os.Getenv("BOB_AUTH_KEY")
```

Comment:
I'm concious that I'm removing the AES encryption here, but near as I can tell it's not used by anything internally, and the ws-manager wraps these ENV vars up in a secret anyway. I'm not sure what this AES gives us now so I removed it to make things as simple and easy as possible. Happy to be corrected if this is still critical or considered a breaking change.